README.md

```diff
@@ -63,6 +65,50 @@ This class simulates the procedure which Xcode uses to invoke xctests.
   * `xctest.stop()`
     * Stop xctest process.
 
+### Mount Developer Image
+
+When using a higher version of iOS devices with a lower version of Xcode or other non-macOS operating systems, most of the functions in `services` or `Xctest` may not be available because the developer image is not mounted. Sometimes it can be solved automatically by opening Xcode and waiting for a while. But more often you need to manually download and mount the developer image as follows：
+
+  1. Download .dmg file with new Xcode(sometimes beta version is needed) from [Apple Developer Downloads Page](https://developer.apple.com/download/more/).
+  2. Unarchive new Xcode from .dmg file without replacing old one.
+  3. Find the folder named after the target iOS version in `(New Xcode.app)/Contents/Developer/Platforms/iPhoneOS.platform/DeviceSupport`. A `DeveloperDiskImage.dmg` and a `DeveloperDiskImage.dmg.signature` should be inside that folder.
```

Comment:
Can we address `AppleTVOS.platform` for tvOS as well? Just a small note should be fine.

Reply:
I've never tried this before.This should be effective on Xcode, but I'm not sure whether this would work using image_mount protocol

Reply:
`ideviceimagemounter` worked for tvos, so maybe this can also work